docs/book/src/topics/spot-instances.md

```diff
@@ -34,4 +34,17 @@ spec:
     spotMarketOptions:
       maxPrice: 0.02 # Price in USD per hour (up to 5 decimal places)
 ```
-> **IMPORTANT NOTE**: The experimental feature `MachinePool` does not support using spot instances as of now.
```

Comment:
This is correct, good to have a note or warning on this here if there is no other warning in the doc.

Reply:
+1

Reply:
A good point. Slightly modified the note from `MachinePool` to `AWSMachinePool` as one variation of machine pool implementation, AWSManagedMachinePool, supports spot instance.